### ISSUE TITLE
"guest" resource should be able to be created from an ISO

### ISSUE THREAD

**Issue by mkaiserincomm** [josenk/terraform-provider-esxi#95]
It would be useful if you could define a guest to be built from an ISO.   I looked in the code and see that this is partially implemented.  Was it abandoned for the sake of time or was it unfeasible for some reason?  I am considering completing this feature but wanted to discuss it to see why you chose not to implement it.

**Comment by josenk** [comment]
Yes, you are correct.   It's already partially implemented...   I didn't see any demand for it and there is already so many other sources to install from, so I never completed it.    A basic 'iso_source' can probably be added without too much difficulty.   If you want to give it a try, you can do a Pull Request.   I'll review it.

Or, you may want to give some of the other choices a try...
